docs/sources/_index.md

```diff
@@ -52,16 +67,21 @@ Beyla offers the following features:
   stable major release.
 - Administrative access rights to execute Beyla.
 
+## Limitations
+
+Beyla has its limitations too. It only provides generic metrics and transaction level trace span information. Language agents and manual instrumentation is still recommended, so that you can specify the granularity of each part of the code to be instrumented, putting the focus on your critical operations.
+
+Another limitation to consider is that Beyla requires elevated privileges; not actually a root user, but at least it has to run with the CAP_SYS_ADMIN capability. If you run the tool as a container (Docker, Kubernetes…), it has to be privileged.
```

Comment:
This has recently changed lately, and might be a more complex:
* Beyla requires CAP_SYS_ADMIN to enable context propagation in Go programs, but can run without it.
* Beyla currently requires other finer-grained capabilities: CAP_DAC_READ_SEARCH, CAP_SYS_PTRACE, CAP_PERFMON, CAP_BPF, CAP_CHECKPOINT_RESTORE, CAP_NET_ADMIN

Reply:
+1 on what Mario said. We should word the text as:

```
While most eBPF programs require elevated privileges, Beyla supports 
specifying finer grained permissions which allow us to run with minimum 
required permissions, such as: CAP_DAC_READ_SEARCH, CAP_SYS_PTRACE, 
CAP_PERFMON, CAP_BPF, CAP_CHECKPOINT_RESTORE, CAP_NET_RAW. 

Some of Beyla's functionality requires further permissions, for example using 
the network observability probes with Linux Traffic Control will require CAP_NET_ADMIN, 
but it's a feature you have to optionally enable. CAP_SYS_ADMIN is only required to 
enable context propagation in Go across multiple nodes, however if this permission 
is not granted, Beyla will gracefully degrade its functionality to support only partial traces.
```